HoloRepositoryDemoApplication/Assets/HoloStorageConnector/Scripts/HoloStorageClient.cs

```diff
@@ -139,8 +147,13 @@ public static IEnumerator GetHologram(Hologram hologram, string HolgramID)
         /// It requires thehologram ID as the parameter 
         /// </summary>
         /// <param name="HologramID">ID of Hologram</param>
-        public static async void LoadHologram(HologramInstantiationSettings setting, string HologramID)
+        public static async void LoadHologram(string HologramID, HologramInstantiationSettings setting = null)
```

Comment:
Can't you do `settings = new HologramInstantiationSettings()` right in the signature?

Reply:
No, it requires the const value, cannot define it as `new HologramInstantiationSettings()`